controllers/release/controller.go

```diff
@@ -82,9 +82,9 @@ func (c *Controller) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 		adapter.EnsureFinalizerIsAdded,
 		adapter.EnsureReleaseExpirationTimeIsAdded,
 		adapter.EnsureTenantPipelineIsProcessed,
+		adapter.EnsureReleaseTenantProcessingIsTracked,
 		adapter.EnsureManagedPipelineIsProcessed,
-		adapter.EnsureReleaseProcessingIsTracked,
-		adapter.EnsureReleaseProcessingResourcesAreCleanedUp,
```

Comment:
@davidmogar FYI I removed this operation. Previously, this operation just removed the finalizer, then the finalize release operation deleted the pipelinerun. We talked about changing it so they used the same code. I tested this, and it broke things because the pipelinerun was being deleted ~as soon as it finished, instead of waiting for the release to be deleted. So, now everything is being deleted when the release is finalized. The only change here is that the roleBinding is now kept around until everything else is deleted (the release and subsequently the two pipelineruns). If this isn't okay, we can talk about it

Reply:
Disregard this. I forgot about the requirement to remove the finalizer from the pipelineRun even before the releaes is called to be deleted. Will update tomorrow